account_financial_report_qweb/report/open_items.py

```diff
@@ -27,6 +27,7 @@ class OpenItemsReport(models.TransientModel):
     company_id = fields.Many2one(comodel_name='res.company')
     filter_account_ids = fields.Many2many(comodel_name='account.account')
     filter_partner_ids = fields.Many2many(comodel_name='res.partner')
+    is_today = fields.Boolean()
```

Comment:
Since this is an internal optimization of the report, I would make this a computed field instead of asking the caller to set it.